public/.htaccess

```diff
@@ -18,7 +18,7 @@ Options All -Indexes
 
 	# Redirect Trailing Slashes...
 	RewriteCond %{REQUEST_FILENAME} !-d
-	RewriteRule ^(.*)/$ /$1 [L,R=301]
+	RewriteRule ^(.*)/$ $1 [L,R=301]
```

Comment:
```suggestion
	RewriteCond %{REQUEST_URI} (.+)/$
      RewriteRule ^ %1 [L,R=301]
```

Reply:
Yeah it's working with your change, I'm commiting this